src/navigator.js

```diff
@@ -324,6 +324,37 @@ $.extend( $.Navigator.prototype, $.EventSource.prototype, $.Viewer.prototype, /*
         this.updateSize();
     },
 
+    /**
+             * Controls the visibility of the navigator element.
+             * This method hides or shows the entire navigator, including its background and borders.
+             * @function
+             * @param {Boolean} visible - True to show the navigator, false to hide it.
+             */
+    setVisible: function (visible) {
+        if (this.element) {
```

Comment:
It might be good to still call the superclass's setVisible, mainly to get the event that it raises. You could end this function with: 

```
        return $.Viewer.prototype.setVisible.apply(this, [visible]);
```